examples/voice_solutions/steering_tts.ipynb

```diff
@@ -0,0 +1,203 @@
+    "\n",
+    "Traditional TTS can specify voices, but not the tone, accent, or any other contextual audio parameters."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from openai import OpenAI\n",
+    "client = OpenAI()\n",
+    "\n",
+    "tts_text = \"\"\"\n",
+    "Once upon a time, Leo the lion cub woke up to the smell of pancakes and scrambled eggs.\n",
+    "His tummy rumbled with excitement as he raced to the kitchen. Mama Lion had made a breakfast feast!\n",
+    "Leo gobbled up his pancakes, sipped his orange juice, and munched on some juicy berries.\n",
+    "\"\"\"\n",
+    "\n",
+    "speech_file_path = \"./sounds/default_tts.mp3\"\n",
+    "response = client.audio.speech.create(\n",
+    "    model=\"tts-1-hd\",\n",
+    "    voice=\"alloy\",\n",
+    "    input=tts_text,\n",
+    ")\n",
+    "\n",
+    "response.write_to_file(speech_file_path)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Chat Completions TTS\n",
+    "\n",
+    "With chat completions, you can give specific instructions before generating the audio. In the following example, we generate a British accent in a learning setting for children. This is particularly useful for educational applications where the voice of the assistant is important for the learning experience."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import base64\n",
+    "\n",
+    "speech_file_path = \"./sounds/chat_completions_tts.mp3\"\n",
+    "completion = client.chat.completions.create(\n",
+    "    model=\"gpt-4o-audio-preview\",\n",
+    "    modalities=[\"text\", \"audio\"],\n",
+    "    audio={\"voice\": \"alloy\", \"format\": \"mp3\"},\n",
+    "    messages=[\n",
+    "        {\n",
+    "            \"role\": \"system\",\n",
+    "            \"content\": \"You are a helpful assistant that can generate audio from text. Speak in a British accent and enunciate like you're talking to a child.\",\n",
+    "        },\n",
+    "        {\n",
+    "            \"role\": \"user\",\n",
+    "            \"content\": tts_text,\n",
+    "        }\n",
+    "    ],\n",
+    ")\n",
+    "\n",
+    "mp3_bytes = base64.b64decode(completion.choices[0].message.audio.data)\n",
+    "with open(speech_file_path, \"wb\") as f:\n",
+    "    f.write(mp3_bytes)\n",
+    "\n",
+    "speech_file_path = \"./sounds/chat_completions_tts_fast.mp3\"\n",
+    "completion = client.chat.completions.create(\n",
+    "    model=\"gpt-4o-audio-preview\",\n",
+    "    modalities=[\"text\", \"audio\"],\n",
+    "    audio={\"voice\": \"alloy\", \"format\": \"mp3\"},\n",
+    "    messages=[\n",
+    "        {\n",
+    "            \"role\": \"system\",\n",
+    "            \"content\": \"You are a helpful assistant that can generate audio from text. Speak in a British accent and speak really fast.\",\n",
```

Comment:
how much more _control_ can you give the model around speed?

Reply:
Honestly not that much. You have to do it in context - e.g. continue to conversation with it rather than on the completions side. This is kinda as much as you can do in speed from the initial convo